src/company_gui.cpp

```diff
@@ -2204,14 +2204,14 @@ static const NWidgetPart _nested_company_widgets[] = {
 							NWidget(NWID_SPACER), SetFill(0, 1),
 						EndContainer(),
 					EndContainer(),
+					/* Multi player buttons. */
 					NWidget(NWID_VERTICAL), SetPIP(4, 2, 4),
-						NWidget(NWID_SPACER), SetMinimalSize(90, 0), SetFill(0, 1),
-						/* Multi player buttons. */
-						NWidget(NWID_HORIZONTAL),
-							NWidget(WWT_EMPTY, COLOUR_GREY, WID_C_HAS_PASSWORD),
+						NWidget(NWID_SPACER), SetFill(0, 1),
+						NWidget(NWID_HORIZONTAL), SetPIP(0, 4, 0),
+							NWidget(WWT_EMPTY, COLOUR_GREY, WID_C_HAS_PASSWORD), SetMinimalSize(10, 0),
```

Comment:
The size of WID_C_HAS_PASSWORD is decided in UpdateWidgetSize(), so SetMinimalSize() is pointless here.

Reply:
You have no idea how much attempts it took to get this to work .. but indeed, it is no longer needed :D Before it fixed the problem where the lock button was in the middle of the whole screen, but it seems something else fixed that :P I am still somewhat confused how the widget system works :D Tnx!